mungegithub/mungers/bulk-lgtm.go

```diff
@@ -57,11 +57,11 @@ func (b *BulkLGTM) Munge(obj *github.MungeObject) {
 		return
 	}
 	glog.V(4).Infof("Found a PR: %#v", *pr)
-	if !*pr.Mergeable {
+	if pr.Mergeable != nil && !*pr.Mergeable {
```

Comment:
It should maybe be:
`if pr.Mergeable == nil || !*pr.Mergeable {`

According to [this](https://developer.github.com/v3/pulls/), `mergeable == nil` means we don't know yet:
> If the value is null, this means that the mergeability hasn't been computed yet, and a background job was started to compute it.